tests/Rector/MethodCall/RenameMethodCallRector/Fixture/fixture4.php.inc

```diff
@@ -12,18 +12,4 @@ class SomeClass5
 }
 
 ?>
------
-<?php
-
-namespace Rector\Tests\Rector\MethodCall\RenameMethodCallRector\Fixture;
-
-use Nette\Utils\Html;
-
-class SomeClass5
-{
-    private function createHtml()
-    {
-    }
-}
-
-?>
+-----
```

Comment:
This line should be remove too. It's separator that indicates to compare to content after it.
Now it's compared to empty string, which fails

```
before|after
stay_the_same
```